septentrion/core.py

```diff
@@ -128,7 +128,9 @@ def get_fixtures_version(
     return version
 
 
-def build_migration_plan(settings: configuration.Settings) -> Iterable[Dict[str, Any]]:
+def build_migration_plan(
+    settings: configuration.Settings, schema_version: Optional[versions.Version] = None
```

Comment:
why is the `schema_version` parameter optional ?

Reply:
At first I thought it was the case, but it later turned out not to be. I'll remove the default at None too, event if it means I have to fix a few tests.